native/src/widget/button.rs

```diff
@@ -234,7 +234,8 @@ where
     }
 
     fn hash_layout(&self, state: &mut Hasher) {
-        std::any::TypeId::of::<Button<'_, (), Renderer>>().hash(state);
+        struct ButtonWidget;
+        std::any::TypeId::of::<ButtonWidget>().hash(state);
```

Comment:
Maybe we could rename this to simply `Marker`, as the module already works as a scope. This should also help us copy-paste these two lines when implementing other widgets.

Reply:
Done :)